realsense_ros2_camera/src/realsense_camera_node.cpp

```diff
@@ -340,6 +340,8 @@ class RealSenseCameraNode : public rclcpp::Node
         "camera/depth/camera_info", 1);
 
       if (_pointcloud) {
+        _align_depth_publisher = this->create_publisher<sensor_msgs::msg::Image>(
+          "/camera/depth/color/aligned_depth_images", 1);
```

Comment:
Any reasons for not to use the same topic name as ROS Realsense? It should be
"/camera/aligned_depth_to_color/camera_info"
"/camera/aligned_depth_to_color/image_raw"

Please try to keep same topic name when porting from ROS to ROS2.